doc/mwc-wallet_coinjoin.md

```diff
@@ -1,16 +1,17 @@
-# MWC-wallet coinjoin pool
+# MWC-wallet CoinJoin Pool
 
-In a context of a low number of on chain transactions, Dandelion is not really effective. To improve privary, MWC-wallet will do a coinjoin at wallet level.  MWC-wallet will talk with other participating mwc-wallets and built a 'multikernel' transaction.
+In a context of a low number of on-chain transactions, Dandelion is not really effective on the Mimblewimble transaction merging/CoinJoin. To improve privacy, MWC-wallet will do a CoinJoin at wallet level, with this design. MWC-wallet will cooperate with other participating MWC-wallets and build a 'multikernel' transaction.
 
-The mwc-wallet coinjoin will take some time to process in order to capture enough transactions from honest wallet participants.
+The MWC-wallet CoinJoin need take more time than Dandelion to process, in order to capture enough transactions from honest wallet participants for this CoinJoin. The time taking of the MWC-wallet CoinJoin will be addressed later.
 
-This method can allow to obfuscate regular transactions, but it's not the primary use case for multiple reasons. In case of Non interactive transactions (NIT) it will be used more easily for regular transactions.
+This method will obfuscate transaction graph, <ins>but it's not the primary use case for multiple reasons</ins> (why? what reason?).
```

Comment:
The workflow normally required output regeneration. If Coinjoin session failed, the outputs need to be regenerated.  It is hard to do with Interactive transactions because slate will need to be send to receiver once more. 
Because of that for Coinjon and IT regular transaction, wallet can't allow session to fail.  It is possible if the wallet will be at the end on the join chain (chain is twice larger than required) and willing to pay the extra fees.  In this case, if next participant failed to process, the wallet will add fees to balance the coinjoin and publish.
So in order to make that work, wallet need to control it's position in the chain and willing to pay extra fees if something will go wrong. Or willing wait for a relatively long time.

I should probably need to add a table that will demonstrate the fees and cost to publish (attack) the coinjoin transaction.